cypress/integration/channel/edit_message_spec.js

```diff
@@ -109,4 +109,46 @@ describe('Edit Message', () => {
             });
         });
     });
+    it('M13542 Open edit modal immediately after making a post', () => {
+        // 1. Login as "user-1" and go to /
+        cy.apiLogin('user-1');
+        cy.visit('/');
+
+        // 2. Post message "Hello"
+        cy.get('#post_textbox').type('Hello').type('{enter}');
+
+        // 3. Post a second message "World!"
+        // 4. Hit 'enter' key and then immediately (as FAST as you can) hit 'arrow-up' to open the edit box
+        cy.get('#post_textbox').type('World!{enter}{uparrow}');
+        cy.get('#edit_textbox').
+            invoke('val').
```

Comment:
I don't think the style checker reads these files (and mabye it wouldn't complain), but I'd prefer to see the dot on the dot notation moved to the next line:
```
cy.get('#edit_textbox')
    .invoke('val')
    .then((val) => { // eslint-disable-line
```

Reply:
@bradjcoughlin done

Reply:
Yup, it's failing due to that.  I suggest to revert the last change as it's the one we have in ESLint rules.

Reply:
@saturninoabril I've reverted the above said changes, please give it a check